imodqgis/widgets/unique_color_widget.py

```diff
@@ -143,10 +182,23 @@ def remove_selection(self) -> None:
             self.table.takeTopLevelItem(self.table.indexOfTopLevelItem(item))
 
     def load_classes(self) -> None:
-        pass
+        path, _ = QFileDialog.getOpenFileName(self, "Load colors", "", "*.txt")
+        with open(path, "r") as file:
+            rgb_values = json.load(file)
+        self.loaded_colors = [QColor(*rgb) for rgb in rgb_values]
+        table_iter = range(self.table.topLevelItemCount())
+
+        for i, color in zip(table_iter, self.loaded_colors):
+            item = self.table.topLevelItem(i)
+            item.setData(1, Qt.ItemDataRole.EditRole, color)
 
     def save_classes(self) -> None:
```

Comment:
Clarify why a JSON file must be used here, rather than the utilities in pseudocolor widget.

Reply:
Done